azext_iot/sdk/iothub/service/operations/query_operations.py

```diff
@@ -41,7 +41,7 @@ def get_twins(
             self, query=None, x_ms_continuation=None, x_ms_max_item_count=None, custom_headers=None, raw=False, **operation_config):
         """Query an IoT Hub to retrieve information regarding device twins using a
         SQL-like language. See
-        https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-query-language
+        https://learn.microsoft.com/azure/iot-hub/iot-hub-devguide-query-language
```

Comment:
If all these generated SDK ones aren't updated at the source (azure-rest-api-specs) next time we regenerate this will be undone

Reply:
I think we can trust the sdk generators to use the correct url. maybe...